CDCgov/cdc-open-viz
# Enhancement [DEV-9961] always show last month

## [DEV-9961](https://websupport-cdc.msappproxy.net/browse/DEV-9961)

Some time the X ticks don't show the last month - The idea is to force it so that it always shows

## Testing Steps
1. Open map with showYearsOnce true (components-templates-chart-axis-labels--abbreviated-dates)
2. Observe that last month has tick and first month of year shows

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)

### Before
<img width="958" alt="Screenshot 2024-12-03 at 4 05 41 PM" src="https://github.com/user-attachments/assets/30768fa9-34b7-429f-9ee7-af1eaaf89819">

### After
<img width="963" alt="Screenshot 2024-12-03 at 4 05 28 PM" src="https://github.com/user-attachments/assets/b5b33173-61c2-47df-9b9e-ffbf0ec5c93b">
